lib/util/template/index.ts

```diff
@@ -189,6 +183,35 @@ const compileInputProxyHandler: ProxyHandler<CompileInput> = {
   },
 };
 
+function getChildEnvFields(): string[] {
+  const childEnv = getChildEnv({});
+  return Object.keys(childEnv);
+}
+
+function getAllowedFieldsList(): Set<string> {
+  const allowedFieldsList = new Set([
+    ...Object.keys(allowedFields),
```

Comment:
create const in module scope for this (perf)

Reply:
I restored the original implementation with consts.

However, I now struggle with two unit tests I added - I don't know how to merge two `env` dictionaries with `const data = { ...GlobalConfig.get(), ...input, env: getChildEnv({}) };`. Probably my misunderstanding of the spread operator `...`. Can you advise please?

Reply:
You can disregard my question, I sorted it out. 

The two tests are still failing, but now I know why. I will post a new question when I'm at my desk.